frameworks/Julia/Jewelia/julia_server.jl

```diff
@@ -14,25 +14,31 @@ end
 
 StructTypes.StructType(::Type{jsonObj}) = StructTypes.Struct()
 
-HTTP.listen("0.0.0.0" , 8080, reuseaddr = true) do http
-    target = http.message.target
-
-    HTTP.setstatus(http, 200)
-    HTTP.setheader(http, "Server" => "Julia-HTTP")
-    HTTP.setheader(http, "Date" => Dates.format(Dates.now(), Dates.RFC1123Format) * " GMT")
-
-    if endswith(target, "/plaintext")
-        HTTP.setheader(http, "Content-Type" => "text/plain")
-        HTTP.startwrite(http)
-        write(http, "Hello, World!")
-
-    elseif endswith(target, "/json")
-        HTTP.setheader(http, "Content-Type" => "application/json")
-        startwrite(http)
-        JSON3.write(http, (;message = "Hello, World!"))
-
-    elseif endswith(target, "/db")
-        HTTP.setheader(http, "Content-Type" => "application/json")
+    function plaintext(req::HTTP.Request)
+        headers = [ "Content-Type" => "text/plain", 
+                    "Server" => "Julia-HTTP",
+                    "Date" => Dates.format(Dates.now(), Dates.RFC1123Format) * " GMT" ]
+    
+        return HTTP.Response(200, headers, body = "Hello, World!")
+    end
+    
+    function jsonSerialization(req::HTTP.Request)
+        headers = [ "Content-Type" => "application/json",
+                    "Server" => "Julia-HTTP",
+                    "Date" => Dates.format(Dates.now(), Dates.RFC1123Format) * " GMT" ]
+        
+        json_string = "{\"Message\": \"Hello, World!\"}"
```

Comment:
In this controller, you have to instantiate an object that then gets serialized.